package.json

```diff
@@ -102,6 +102,7 @@
     "rollup-plugin-node-resolve": "^5.1.0",
     "rollup-plugin-terser": "^5.0.0",
     "sass-loader": "^8.0.0",
+    "storybook-chromatic": "^3.1.0",
```

Comment:
The `chromaui/actions` Github Action normally takes care of this dependency. Github Actions can provide their own `node_modules` dependencies which is cool. This will be required until we get Github Actions back.